utils/cups-browsed.c

```diff
@@ -7807,7 +7807,35 @@ gboolean update_cups_queues(gpointer unused) {
 #ifdef HAVE_CUPS_1_6
       /* Check whether there is a temporary CUPS queue which we would
          overwrite */
-      dest = cupsGetNamedDest(http, p->queue_name, NULL);
+      ipp_t *response =  NULL;
```

Comment:
You should set "dest = NULL" before this line, so that if the get-printer-attributes request is without result or with bad result that dest is not on a random value or on the value of the previous CUPS queue updated, so that nothing wrong can happen on the "if (dest)" below.

Reply:
Thank You!
I have assigned dest = NULL.